notify_listener.go

```diff
@@ -94,8 +94,13 @@ func (l *NotifyListener) ListenForChanges(ctx context.Context) (chan *Changeset,
 	l.logger.Info("Starting notify listener for `warp_pipe_new_changeset`")
 	l.store = store.NewChangesetStore(l.conn)
 
+	err := loadColumnTypesPGX(l.conn)
+	if err != nil {
+		l.logger.WithError(err).Fatal("failed to load the column types")
```

Comment:
We _really_ shouldn't use `Fatal()` ever since no cleanup tasks are run.

Reply:
I just followed the already existing pattern. Do you want me add another commit placing all log fatals on the err channel?